understanding/20/pause-stop-hide.html

```diff
@@ -110,58 +88,58 @@ <h2>Intent of Pause, Stop, Hide</h2>
          moves the focus away) would not be considered a "mechanism for the user to pause"
          because it makes the page unusable in the process and would not meet this SC.
       </p>
-      
-      <p>It is important to note that the terms "blinking" and "flashing" can sometimes refer
-         to the same content.
-      </p>
-      
+
+      <div class="note">
+         <p>This Success Criterion is specifically concerned with moving, blinking, scrolling, and
+            auto-updating visual content. For audio content that starts automatically, refer to <a href="audio-control">1.4.2 Audio Control</a>.</p>
+         <p>Moving, blinking, scrolling content that starts automatically not as a result of explicit *activation* (such as activating a button),
+            but because of a more general user interaction (such as focusing/hovering over an element, or scrolling the page), and doesn't provide
+            a way to Pause, Stop, or Hide, will fail this Criterion, and potentially
```

Comment:
> "...and doesn't provide a way tp Pause, Stop, or Hide, will fail..."


This seems a bit clumsy. Maybe more compact and easier to write "...without providing a way to Pause, Stop, or Hide, will fail..."

Reply:
turned this whole paragraph around a bit to make it clearer, hopefully